contentcuration/contentcuration/models.py

```diff
@@ -131,6 +132,17 @@ def __unicode__(self):
     def delete(self):
         # Remove any invitations associated to this account
         self.sent_to.all().delete()
+
+        # Delete channels associated with this user (if user is the only editor)
+        for c in self.editable_channels.all():
```

Comment:
Could we do these two in a fixed number of queries? Maybe something like:
```
self.editable_channels.annotate(num_editors=Count("editors")).filter(num_editors=1).delete()
```

And similarly for channel sets below?

Reply:
Seeing an issue where Count always returns 1 even if multiple editors are available on the channel. Going to see if I can get SQCount to work instead